packages/modal/src/Modal/Modal.stories.tsx

```diff
@@ -34,7 +34,7 @@ type Story = StoryObj<typeof TelegraphModal.Root>;
 export const Modal: Story = {
   render: () => {
     // eslint-disable-next-line react-hooks/rules-of-hooks
-    const [open, setOpen] = React.useState(true);
+    const [open, setOpen] = React.useState(false);
```

Comment:
Changed these to false by default so they don't display over all of the storybook content now that they're in portals.